### ISSUE TITLE
Name conflict with Data.Foldable on GHC 7.10+

### ISSUE THREAD

**Issue by identicalsnowflake** [agda/agda#2016]
```
src/full/Agda/TypeChecking/Monad/Base.hs:1337:46:
    Ambiguous occurrence ‘null’
    It could refer to either ‘Agda.Utils.Null.null’,
                             imported from ‘Agda.Utils.Null’ at src/full/Agda/TypeChecking/Monad/Base.hs:92:1-22
                             (and originally defined at src/full/Agda/Utils/Null.hs:49:3-20)
                          or ‘Data.Foldable.null’,
                             imported from ‘Data.Foldable’ at src/full/Agda/TypeChecking/Monad/Base.hs:40:1-20
cabal: Error: some packages failed to install:
Agda-2.6.0 failed during the building phase. The exception was:
ExitFailure 1
make: *** [install-bin] Error 1
```

Breaks on CI with GHC 8.x and for me locally on 7.10.3.

**Comment by asr** [comment]
This issue is fixed in [this](https://github.com/agda/agda/pull/2015) PR. @Blaisorblade will merge the PR.

**Comment by identicalsnowflake** [comment]
Ok thanks I'll use that for now.

**Comment by asr** [comment]
Fixed by 1bdc79d.
